src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagInfoTest.java

```diff
@@ -362,6 +362,7 @@ public void testSerialField() {
                 .that(JavadocTagInfo.SERIAL_FIELD.isValidOn(ast))
                 .isFalse();
 
+        astChild2.setText("ObjectStreamField");
```

Comment:
We do not allow pure unit testing anymore.
Please kill mutation by extension of some input file for some specific Check.